reframe/core/exceptions.py

```diff
@@ -50,7 +77,7 @@ class ReframeFatalError(ReframeBaseError):
 
 
 class ReframeSyntaxError(ReframeError):
-    '''Raised when the syntax of regression tests is not correct.'''
+    '''Raised when the syntax of regression tests iscorrect.'''
```

Comment:
```suggestion
    '''Raised when the syntax of regression tests is not correct.'''
```

Reply:
I wanted to write "is incorrect" but I messed it!